commands/src/main/java/org/wildfly/extras/creaper/commands/logging/Level.java

```diff
@@ -0,0 +1,35 @@
+package org.wildfly.extras.creaper.commands.logging;
+
+public enum Level {
+
+    ALL("ALL"),
+    FINEST("FINEST"),
+    FINER("FINER"),
+    TRACE("TRACE"),
+    DEBUG("DEBUG"),
+    FINE("FINE"),
+    CONFIG("CONFIG"),
+    INFO("INFO"),
+    WARN("WARN"),
+    WARNING("WARNING"),
+    ERROR("ERROR"),
+    SEVERE("SEVERE"),
+    FATAL("FATAL"),
+    OFF("OFF");
```

Comment:
Make sure these are sorted. I think `FINER` sorts _after_ `TRACE` (actually, I think that `FINEST` and `TRACE` are synonyms, `FINER` and `DEBUG` too, but please confirm first).